packages/core/ui-settings/core-ui-settings-server-internal/src/create_or_upgrade_saved_config/get_upgradeable_config.test.ts

```diff
@@ -46,6 +46,41 @@ describe('getUpgradeableConfig', () => {
     expect(result).toEqual(savedConfig);
   });
 
+  it('uses the latest config when multiple are found', async () => {
+    const savedObjectsClient = savedObjectsClientMock.create();
+    savedObjectsClient.find.mockResolvedValue({
+      saved_objects: [
+        { id: '7.2.0', attributes: 'foo' },
```

Comment:
Could there ever be a scenario where we have an invalid version value in the `id` field? If so, would be nice to have test coverage for it. If not, ignore this!

Reply:
In theory, there shouldn't, but life alway finds a way, so I guess adding a test wouldn't hurt